src/components/MaterialUI/Inputs/DatePicker.js

```diff
@@ -133,10 +137,25 @@ const WrappedDatePicker = ({
 	const classes = useStyles({ readOnly });
 	const startDate = value ? new Date(value) : null;
 	const disabledCls = classNames({ [classes.disabled]: props.disabled });
-
+	const localizedTimeZoneName = useSelector(namedLookupLocalizedSelector("customer", "TimeZone", timePickerTimeZone));
+	const setLocalZone = (date, timezone) => {
+		const dateWithoutZone = moment.tz(date, timezone).format("YYYY-MM-DDTHH:mm:ss.SSS");
+		const localZone = moment(dateWithoutZone).format("Z");
+		const dateWithLocalZone = [dateWithoutZone, localZone].join("");
+		return new Date(dateWithLocalZone);
+	}
+	  
+	const setOtherZone = (date, timezone) => {
+		console.log(timezone);
```

Comment:
remove console.log
and move this methods to timezoneHelper and cover with tests